agenta-backend/agenta_backend/services/evaluation_service.py

```diff
@@ -106,6 +146,41 @@ async def _fetch_evaluation_scenario_and_check_access(
     return evaluation_scenario
 
 
+async def create_new_bulk_evaluation(app, payload, **user_org_data: dict):
+    status = "EVALUATION_CREATED"
+    user = await get_user(user_uid=user_org_data["uid"])
+    current_time = datetime.utcnow()
+
+    testset = await db_manager.fetch_testset_by_id(testset_id=payload.testset_id)
+
+    evaluation_type_settings = EvaluationTypeSettings(
```

Comment:
not fan of this. Especially when we start having many eval methods. this would grow very large